src/index.html

```diff
@@ -1,114 +1,113 @@
 <!DOCTYPE html>
-<html>
-<head>
-  <link href="https://fonts.googleapis.com/css?family=Noto+Sans+SC:300,400,700&display=swap" rel="stylesheet">
-  <title>FitLit • Activity Tracker</title>
-  <script src="https://cdn.jsdelivr.net/npm/chart.js@2.8.0"></script>
-</head>
-
-<body>
-  <nav>
-    <div>
-      <h2>FitLit</h2>
-      <h6 id="date"></h6>
-    </div>
-  </nav>
-  <header>
-    <div id="name">
-    </div>
-    <ul>
-      <li>Address: <span id="address"></span></li>
-      <li>Email: <span id="email"></span></li>
-      <li>Stride Length: <span id="strideLength"></span></li>
-      <li>Daily Step Goal: <span id="stepGoal"></span></li>
-      <li>Friends: <span id="friend-list"></span></li>
-    </ul>
-  </header>
-  <main>
-    <section class="main__section--steps main__sections">
-      <h3 class="section__h3--title">Steps</h3>
-      <section class="steps--container widget--container">
-        <article class="steps__article--1 article--widget">
-          <h5>DAILY STEP GOAL</h5>
-          <canvas id="step-goal-chart"></canvas>
-          <p id="step-compare">
-          </p>
-        </article>
-        <article class="steps__article--2 article--widget">
-          <h5>WEEKLY FRIEND SCOREBOARD</h5>
-          <ul id="friend-weekly-steps"></ul>
-        </article>
-        <article class="steps__article--3 article--widget">
-          <h5>TRENDS</h5>
-          <img class="trend-icon" src="./images/trend-icon.png">
-          <div id="step-trends"></div>
-        </article>
+<html lang="en">
```

Comment:
Awesome that it took something to simple to amp up our accessibility score. I don't think I've used the `lang` tag, this is great to note down

Reply:
That's great!  I was doing some reading about this, and it looks like you always only use 2 letters (which you did) so nice work!